ts/features/euCovidCert/screens/ko/EuCovidCertNotFoundKoScreen.tsx

```diff
@@ -1,22 +1,120 @@
 import * as React from "react";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
-import { H1 } from "../../../../components/core/typography/H1";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
+import I18n from "../../../../i18n";
 import { GlobalState } from "../../../../store/reducers/types";
 import { BaseEuCovidCertificateLayout } from "../BaseEuCovidCertificateLayout";
+import CopyButtonComponent from "../../../../components/CopyButtonComponent";
+import doubtImage from "../../../../../img/pictograms/doubt.png";
+import { InfoScreenComponent } from "../../../../components/infoScreen/InfoScreenComponent";
+import { H4 } from "../../../../components/core/typography/H4";
+import { EUCovidCertificateAuthCode } from "../../types/EUCovidCertificate";
+import { euCovidCertCurrentSelector } from "../../store/reducers/current";
+import WorkunitGenericFailure from "../../../../components/error/WorkunitGenericFailure";
+import { mixpanelTrack } from "../../../../mixpanel";
+import { confirmButtonProps } from "../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import { openWebUrl } from "../../../../utils/url";
+import { euCovidCertificateUrl } from "../../../../urls";
+import i18n from "../../../../i18n";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
-const EuCovidCertNotFoundKoScreen = (_: Props): React.ReactElement => (
-  <BaseEuCovidCertificateLayout
-    testID={"EuCovidCertNotFoundKoScreen"}
-    content={<H1>TMPEuCovidCertNotFoundKoScreen</H1>}
-  />
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  }
+});
+
+const CopyWithTitleItem: React.FC<{
```

Comment:
If the left column has a long text we'll have a screen width overflow

<img src="https://user-images.githubusercontent.com/822471/120918558-4ee5df00-c6b5-11eb-9cd4-5187fbd3a99d.png" height="600"/>

To simulate this scenario add a long test [here](https://github.com/pagopa/io-dev-api-server/blob/b19ed39c83d30725dcc11535d3af30c5e844343d/src/routers/message.ts#L154) in the dev-server

Reply:
Thank you, fixed in  [c1b9ab9](https://github.com/pagopa/io-app/pull/3091/commits/c1b9ab98a7e1a82f414fcbd9c0e289f82c4e07c7)
<img src="https://user-images.githubusercontent.com/11773070/120981766-49050200-c778-11eb-9b74-5643272d80b5.png" width=300>